src/App.tsx

```diff
@@ -101,6 +104,17 @@ const Content = () => {
     }
   }, [dispatch]);
 
+  useEffect(() => {
+    appQueryClient.prefetchQuery({
+      queryKey: ['transferEligibleChains', { skipClient }],
```

Comment:
I don't think this is a valid query key. They need to be basic objects and serializable. I think we should make a helper that memoizes a mapping from this thing to a uuid that churns when the client reference ever changes.

Reply:
ooo didn't know that. that's fine we can just have the queryFn's expect a skipClient as a parameter. no biggie.

also i like that idea. should be easy enough to do this by returning a uuid in the `useMemo` that defines the `skipClient` const